tests/richenum/test_rich_enums.py

```diff
@@ -58,6 +58,11 @@ def test_enums_iterate_through_members(self):
         members = set(e for e in Vegetable)
         self.assertEqual(members, set((Vegetable.OKRA, Vegetable.BROCCOLI)))
 
+    def test_lookup_error_type(self):
+        with self.assertRaises(EnumLookupError) as error:
+            Vegetable.from_canonical('parsnip')
+        self.assertIsInstance(error.exception, LookupError)
```

Comment:
You could change this whole test to just do:
`self.assertTrue(issubclass(EnumLookupError, LookupError))`

I think all of the other cases are covered by the `self.assertRaises(EnumLookupError)` check in other tests.

IIRC, we have other tests to ensure that each RichEnum specific  `LookupError` inherits from `EnumLookupError`